npm/src/debug-service/worker-node.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This module supports running the qsharp compiler in a Node.js worker thread. It should be
```

Comment:
Again, copy-paste comment that still refers to the compiler

I do understand all the webworker stuff is copy-paste, and this is really giving me an incentive to refactor right now so that we don't have to duplicate 4 files of boilerplate every time we add a new service :)